lib/components/common/timeline/list_timeline/cubit/list_timeline_cubit.dart

```diff
@@ -7,6 +7,9 @@ class ListTimelineCubit extends TimelineCubit {
     required this.listId,
   }) {
     loadInitial();
+    filter = TimelineFilter.fromJsonString(
+      app<TimelineFilterPreferences>().listTimelineFilter,
+    );
   }
```

Comment:
We also need to override the `persistFilter` method to save the filter into the shared preferences when it's updated

```dart
@override
void persistFilter(String encodedFilter) {
  app<TimelineFilterPreferences>().listTimelineFilter = encodedFilter;
}
```

Reply:
added 👍🏾